### ISSUE TITLE
Does not tag members who has not a username

### ISSUE THREAD

**Issue by emreyilmaz61** [Matt0550/TagEveryoneTelegramBot#7]
Please check. Bot is only tagging users with username.

**Comment by Matt0550** [comment]
> Please check. Bot is only tagging users with username.

Yes, at the moment the bot only tags people who have a username. I put this issue as an enchantment and I hope to solve it asap.
